### DISCUSSION TITLE
Wordpress

### DISCUSSION THREAD

**Discussion by danniehansen** [brefphp/bref#815] [General]
I've been working on a serverless.yml configuration file to get starting on hosting a website running Wordpress on Lambda using Bref that will auto-configure VPC, Database, Security groups etc.

I was wondering if anyone else taken on this adventure or if there is already resources available for this? Personally I have a few websites I think could benefit hugely running on Lambda. Not only billing wise but also with the added security of a read-only file system in case it ever got hacked.

Managed to get all responding correctly with ~120ms response times. Still working on setting up auto-configured CloudFront on top of it for caching and for serving static files.

Just wanted to touch base in case someone already have a blog post out and about explaining the process of running Wordpress on Bref.

**Comment by kevincerro** [comment]
Hi @danniehansen

Please can you publish the `serverless.yml` file?
It will help other people like me to have an idea about how to configure these things.
Also this can allow other more experienced people to help.

Thank you.

**Reply by danniehansen** [reply]
Hi @kevincerro 

Sorry for not responding. The email notification from your message must have gotten list in my inbox.
Unfortunately I've since laying this project on the shelf for a while cleaned up my desktop and accidentally removed the project folder.
From what I remember routing was the hardest. Deploying the wordpress installation worked fine & was rather simple. But Bref has a single handler that receives requests - so I had to write my own request handler to support dynamic routes and static .php routes.

I'll follow up here when I pick up the project again.
